Description: Re-add "GEM" to the dri renderer string of the intel driver.
 Removing it breaks KDE's detection for the blur effect.
 Should be dropped with mesa 7.11
Bug-Ubuntu: https://bugs.launchpad.net/bugs/753370

--- a/src/mesa/drivers/dri/intel/intel_context.c
+++ b/src/mesa/drivers/dri/intel/intel_context.c
@@ -62,6 +62,10 @@
 #endif
 
 
+#define DRIVER_DATE                     "20100330 DEVELOPMENT"
+#define DRIVER_DATE_GEM                 "GEM " DRIVER_DATE
+
+
 static const GLubyte *
 intelGetString(struct gl_context * ctx, GLenum name)
 {
@@ -178,7 +182,7 @@
          break;
       }
 
-      (void) driGetRendererString(buffer, chipset, "", 0);
+      (void) driGetRendererString(buffer, chipset, DRIVER_DATE_GEM, 0);
       return (GLubyte *) buffer;
 
    default:
